### ISSUE TITLE
Stable release accidentally marked pre-release?

### ISSUE THREAD

**Issue by cpsauer** [clangd/clangd#746]
I'd noticed that the 12.0.0 release this morning, described as stable, is also marked as pre-release. Thought I should give a friendly heads up in case that was unintentional. (If intentional, apologies for the false positive.)

Thanks for a most wonderful piece of software :) I'm excited to use 12.0!
Chris

**Comment by kirillbobyrev** [comment]
Hi Chris! Yes, this is intentional: we wanted to get a cherry-pick into 12.0.0 but were sadly late for the release (I know the schedule is off but still :disappointed:). I triggered the release but then realized the cherry-pick situation and so I marked it "pre-release".

I don't know if we will delete this release and reuse the name (most users won't know the difference) or simply make something like 12.0.0-1/12.0.0-patch and mark it as the release one but we're expecting the updated version within a week or so.

The cherry-pick is related to the [remote index](https://clangd.llvm.org/remote-index.html) feature but it doesn't change the feature functionality, we just want to make the configuration safer.

Thanks for noticing and bringing it up: this time it was intentional but in other cases it might not be :smile: So it certainly helps when someones raises the awareness, thank you for that and happy coding!
